### ISSUE TITLE
Low serum aspartate aminotransferase

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5947]
**Preferred term label:** Low serum aspartate aminotransferase

**Synonyms** Low AST; Aspartate aminotransferase decreased

**Definition (free text, please give PubMed ID)** An abnormally low concentration in the circulation of aspartate aminotransferase (AST)., PMID:30320474

**Parent term (use hpo.jax.org/app)** Elevated hepatic transaminase HP:0002910

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
There is no medically relevant "low" AST level -- basically, the test is whether AST is increased or not.
